### ISSUE TITLE
Suck at keyboard "Enter your name" prompts

### ISSUE THREAD

**Issue by Theguysayhi** [uyjulian/renpy-switch#21]
Hi there,

When a prompt for keyboard input appears, the switch keyboard appears as normal, however, it refuses to go away even after I finish typing and hit enter.

This causes a soft lock for many games.

**Comment by ObscenityIB** [comment]
#20 

Need an example for running the libnx bind for the keyboard applet

it would be required in this situation

**Comment by Theguysayhi** [comment]
```
$ player.name = renpy.input(_("What's your name?"), length=13) or _("Hero")
$ player.name.strip()
$ save_name = copy.deepcopy(player.name)
```
 
Here's the code in question.
After the input is provided via the keyboard applet, it does not go away.

Sorry for the late reply.

**Comment by ObscenityIB** [comment]
yeah we just have to wait for keyboard applet binding examples, then it can be possible
